jsonnet/rules.jsonnet

```diff
@@ -196,6 +196,17 @@
               severity: 'warning',
             },
           },
+          {
+            expr: ' sum(increase(kube_pod_container_status_restarts_total[15m]) and on (namespace, container, pod) kube_pod_container_status_last_terminated_reason{reason="OOMKilled"} == 1 ) > 5',
```

Comment:
you might want this to avoid many-to-many errors:

```
sum(max by(namespace, container, pod) (increase(kube_pod_container_status_restarts_total[15m])) and max by (namespace, container, pod) (kube_pod_container_status_last_terminated_reason{reason="OOMKilled"}) == 1) > 5
```

Reply:
Done, thanks!